### PR TITLE
Fix typo row-level-security.mdx

### PR DESCRIPTION
**Pull request by m-tse**
The sentence was missing the noun.

## What kind of change does this PR introduce?

docs update

## What is the current behavior?

Please link any relevant issues here.

## What is the new behavior?

Feel free to include screenshots if it includes visual changes.

## Additional context

Add any other context or screenshots.

### CODE REVIEW

**Review by 0xflotus — APPROVED** [review]

**Comment by fsansalvadore** [comment]
@m-tse it looks like you need to run prettier. 

``npm i && npm run format``

**Comment by m-tse** [comment]
ran npm format, and it changed a lot of files which I never touched. It's weird that non formatted code made it into master.

**Comment by charislam** [comment]
TYSM! Do you mind reverting unrelated changes and keeping it to the typo fix for this PR?

**Comment by charislam** [comment]
something odd with our prettier config, what is being run on local doesn't match what is being run on CI

forgive me for taking over your PR to figure it out 😅 the change looks good and I'll merge once I can figure out what on earth is going on with prettier

**Comment by m-tse** [comment]
Yes, please take the PR off my hands, I just wanted to fix a one word typo I found!

File: .prettierrc [hunk 1]
```diff
@@ -7,5 +7,13 @@
...
-  "plugins": ["prettier-plugin-sql-cst"]
+  "plugins": ["prettier-plugin-sql-cst"],
+  "overrides": [ 
+	  {
+		  "files": "**/tsconfig.json",
+		  "options": {
+			  "trailingComma": "none"
+		  }
+	  }
+  ]
```
Comment (charislam) [inline .prettierrc:18]:
```suggestion
  "plugins": ["prettier-plugin-sql-cst"]
```

**Review by charislam — APPROVED** [review]

**Pull request merged**